v5_ws_public_ticker.go

```diff
@@ -0,0 +1,185 @@
+package bybit
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/gorilla/websocket"
+)
+
+// SubscribeTicker :
+func (s *V5WebsocketPublicService) SubscribeTicker(
+	key V5WebsocketPublicTickerParamKey,
+	f func(V5WebsocketPublicTickerResponse) error,
+) (func() error, error) {
+	if err := s.addParamTickerFunc(key, f); err != nil {
+		return nil, err
+	}
+	param := struct {
+		Op   string        `json:"op"`
+		Args []interface{} `json:"args"`
+	}{
+		Op:   "subscribe",
+		Args: []interface{}{key.Topic()},
+	}
+	buf, err := json.Marshal(param)
+	if err != nil {
+		return nil, err
+	}
+	if err := s.writeMessage(websocket.TextMessage, buf); err != nil {
+		return nil, err
+	}
+	return func() error {
+		param := struct {
+			Op   string        `json:"op"`
+			Args []interface{} `json:"args"`
+		}{
+			Op:   "unsubscribe",
+			Args: []interface{}{key.Topic()},
+		}
+		buf, err := json.Marshal(param)
+		if err != nil {
+			return err
+		}
+		if err := s.writeMessage(websocket.TextMessage, []byte(buf)); err != nil {
+			return err
+		}
+		s.removeParamTickerFunc(key)
+		return nil
+	}, nil
+}
+
+// V5WebsocketPublicTickerParamKey :
+type V5WebsocketPublicTickerParamKey struct {
+	Symbol SymbolV5
+}
+
+// Topic :
+func (k *V5WebsocketPublicTickerParamKey) Topic() string {
+	return fmt.Sprintf("%s.%s", V5WebsocketPublicTopicTicker, k.Symbol)
+}
+
+// V5WebsocketPublicTickerResponse :
+type V5WebsocketPublicTickerResponse struct {
+	Topic     string                      `json:"topic"`
+	Type      string                      `json:"type"`
+	TimeStamp int64                       `json:"ts"`
+	CrossSeq  int64                       `json:"cs,omitempty"`
+	Id        string                      `json:"id,omitempty"`
```

Comment:
nits:

These do not seem to be common, so it would be good to have them on each result side.